lighttpd/datadog_checks/lighttpd/lighttpd.py

```diff
@@ -89,7 +89,11 @@ def check(self, instance):
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK, tags=service_check_tags)
 
         headers_resp = r.headers
-        server_version = self._get_server_version(headers_resp)
+        full_version, server_version = self._get_server_version(headers_resp)
+
+        if full_version:
+            self.set_metadata('version', full_version)
```

Comment:
let's log a message in the scenario we don't submit a version

Reply:
Updated!